client/src/components/LevelLayout.tsx

```diff
@@ -50,6 +56,19 @@ const LevelLayout: React.FC<Props> = ({
         {showInput && <StepValidation />}
         {children}
       </Container>
+      <Flex>
+        <Spacer />
+        <Box
+          display="flex"
+          alignItems="center"
+          bg="gray.200"
+          p={4}
+          borderWidth="1px"
+          borderRadius="lg"
+        >
+          {showAttempts && <Text>Remaining Attempts: {game.attempts} /3 </Text>}
```

Comment:
I am pretty sure `game.attempts` is starting at 0 and increments. If we want to display remaining attempts, then you should take the total number of attempts and subtract `game.attempts`.

Reply:
Resolve on last push